CONTRIBUTING.md

```diff
@@ -55,9 +55,31 @@ using Docker.
 
 
 ### Running Tests
```

Comment:
Maybe worth adding something about what tests we would expect someone to run. E.g I wouldn't expect anyone contributing to run the uptime/scale tests themselves, but we would expect them to run the unit/integration tests. Maybe the acceptance tests?